src/AzurePipelinesToGitHubActionsConverter.Tests/StepsTests.cs

```diff
@@ -122,6 +122,40 @@ public void BashIndividualStepTest()
             Assert.AreEqual(expected, gitHubOutput.actionsYaml);
         }
 
+        [TestMethod]
```

Comment:
@maximivanov, how does this look?

Reply:
Thanks for looking into that! Couple things to check:

`restoreKeys` should be `restore-keys` in Github Actions.
`runner.os` is how you reference the OS in Github Actions.

I will give it a proper test later today.

Reply:
Good catch on restore-keys, and runner.os is an oversight. I've fixed them both and I'm going to approve this and push it to the website.